packages/jss-starter-kit/readme.md

```diff
@@ -1,67 +1,19 @@
-# jss-starter-kit:🚰 the whole kitchen sink
+# jss-starter-kit
 
-[![Gitter](https://badges.gitter.im/JoinChat.svg)](https://gitter.im/cssinjs/lobby)
+> A bundle to get you started playing with JSS. Not optimized for production deployment.
 
-The `jss-starter-kit` combines all of the other [packages](https://github.com/cssinjs/jss/tree/master/packages) into a single bundle, to make it easy to import directly into a playground like Codepen, or a local HTML file.
+See our website [jss-starter-kit](https://cssinjs.org/jss-starter-kit?v=v10.0.0-alpha.8) for more information.
 
-:warning: This bundle includes every JSS package, unminified. It's great for experimenting, but not suitable for production deployment.
+## Install
 
-## Quick start
+Using npm:
 
-There's an instance of JSS already set up with all of the default plugins, exported as `jss`.
```

Comment:
Just realized we have no good place where all exports from the starter kit are documented. We should probably create a docs/jss-starter-kit.md and list them all there